test/quantization/test_quantize_script.py

```diff
@@ -50,1178 +50,1345 @@
 import itertools
 import unittest
 
-class TestQuantizeScript(JitTestCase):
-    def test_insert_observers(self):
-        class M(torch.nn.Module):
+class TestQuantizeScriptPasses(JitTestCase):
```

Comment:
Can we call this TestQuantizeScriptJitPass instead? Or just TestQuantizedJitPasses

Reply:
sure, will rename to JitPasses